packages/frontend/src/components/VisualizationConfigs/TableConfigPanel/index.tsx

```diff
@@ -14,13 +16,23 @@ const TableConfigPanel: React.FC = () => {
     const disabled = !resultsData;
 
     return (
-        <Popover2
-            disabled={disabled}
-            position="bottom"
-            content={
+        <Popover {...COLLAPSABLE_CARD_POPOVER_PROPS} disabled={disabled}>
+            <Popover.Target>
+                <Button
+                    {...COLLAPSABLE_CARD_BUTTON_PROPS}
+                    disabled={disabled}
+                    rightIcon={
+                        <MantineIcon icon={IconChevronDown} color="gray" />
+                    }
+                >
+                    Configure
+                </Button>
+            </Popover.Target>
+
+            <Popover.Dropdown>
                 <Box
                     w={320}
-                    p="sm"
+                    p={0}
                     sx={{
                         // FIXME: remove after Blueprint migration is complete
```

Comment:
could you check if this is necessary now? 🤔

Reply:
The next review removes this too :)